specs/commandline/inspect.md

```diff
@@ -0,0 +1,269 @@
+            ├── io.cncf.notary.signingAgent: notation/1.0.0
+        ├── certificates
+            ├── SHA1 fingerprint: 2f1cc5b8455381cdefac83b4bd305b789cc9c16e
+                ├── issued to: Microsoft Root Certificate Authority 2010
+                ├── issued by: Microsoft Root Certificate Authority 2010
+                ├── expiry: Sat Jun 23 22:04:01 2035
+            ├── SHA1 fingerprint: 8BFE3107712B3C886B1C96AAEC89984914DC9B6B
+                ├── issued to: Microsoft Code Signing PCA 2010
+                ├── issued by: Microsoft Root Certificate Authority 2010
+                ├── expiry: Sun Jul 06 20:50:17 2025
+        └── signed attribute 
+            ├── media type: application/vnd.oci.image.manifest.v1+json
+            ├── digest: sha256:ca5427b5567d3e06a72e52d7da7dabfac484efe37a5380ee9088f7ace2eaab9
+            └── size: 16724
+    ├── sha256:bbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbb
+        ├── signing algorithm: RSASSA-PSS-SHA-256
+        ├── signed attributes
+            ├── content type: application/vnd.cncf.notary.payload.v1+json
+            ├── signing scheme: notary.signingAuthority.x509
+            ├── signing time: Fri Jun 23 22:04:01 2023
+            ├── expiry: Sat Jun 29 22:04:01 2024
+            ├── io.cncf.notary.verificationPlugin: com.example.nv2plugin     
+        ├── unsigned attributes
+            ├── io.cncf.notary.timestampSignature: <Base64(TimeStampToken)> 
+            ├── io.cncf.notary.signingAgent: notation/1.0.0   
+        ├── certificates
+            ├── SHA1 fingerprint: 2f1rr5b8455381frdajc83b4bd305b743cc9513u
+                ├── issued to: Microsoft Root Certificate Authority 2010
+                ├── issued by: Microsoft Root Certificate Authority 2010
+                ├── expiry: Sat Jun 23 22:04:01 2035
+            ├── SHA1 fingerprint: 8BFE3107712B3C886B1C96AAEC89984914DC9B6B
+                ├── issued to: Microsoft Code Signing PCA 2010
+                ├── issued by: Microsoft Root Certificate Authority 2010
+                ├── expiry: Sun Jul 06 20:50:17 2025
+        └── signed artifact 
+            ├── media type: application/vnd.oci.image.manifest.v1+json
+            ├── digest: sha256:ca5427b5567d3e06a72e52d7da7dabfac484efe37a5380ee9088f7ace2eaab9
+            └── size: 16724
+```
+## Inspect signatures on the supplied OCI artifact with an example of JSON Output
+
+```shell
+notation inspect localhost:5000/net-monitor@sha256:b94d27b9934d3e08a52e52d7da7dabfac484efe37a5380ee9088f7ace2efcde9 -o json
+```
+
+An example output:
+```jsonc
+{
+  "signatures": [
+    {
+      "digest": "sha256:73c803930ea3ba1e54bc25c2bdc53edd0284c62ed651fe7b00369da519a3c333",
+      "signingAlgorithm": "RSASSA-PSS-SHA-256",
+      "signedAttributes": {
+        "contentType": "application/vnd.cncf.notary.payload.v1+json",
+        "signingScheme": "notary.default.x509",
+        "signingTime": "Sun Feb 06 20:50:17 2022",
+        "expiry": "Sun Feb 06 20:50:17 2023",
```

Comment:
The date time in the _JSON_ view should follow RFC 3339 like `2022-02-06T20:50:17Z`. Otherwise, it is hard for scripts or other programs to consume those dates.